sed/calibrator/delay.py

```diff
@@ -25,22 +28,102 @@ class DelayCalibrator:
     def __init__(
         self,
         config: dict = None,
-    ):
+    ) -> None:
         """Initialization of the DelayCalibrator class passes the config.
 
         Args:
             config (dict, optional): Config dictionary. Defaults to None.
         """
-        if config is None:
-            config = {}
-
-        self._config = config
+        self._config: dict = config or {}
```

Comment:
This kind of construct has raised an error in the tests, at least for list (with the sector delays). Not sure when and why this is a problem. I'd rather add one more line and make sure the code works.

Reply:
never gave me problems, not even in tests run locally. But I'll change it...